widget/embedded/src/components/TokenList/TokenList.styles.ts

```diff
@@ -48,13 +48,6 @@ export const tokenAddressStyles = css({
 export const tokenWithoutNameStyles = css({});
 export const usdValueStyles = css();
 
-export const Container = styled('div', {
```

Comment:
fixed.
I reverted this in TokensList

Reply:
I revert back `Container` . Yes, I just added `justifyContent`, because I wanted to display the not found message in the middle of the screen when the token was not found

Reply:
You only renamed the component. please also move it on top of `export const Title = styled('div', {` as well.

You can see your change on Github to see where it was and where you moved it.

Reply:
fixed